pkg/operator/controller/ingress/internal_service.go

```diff
@@ -36,7 +40,11 @@ func (r *reconciler) ensureInternalIngressControllerService(ic *operatorv1.Ingre
 		log.Info("created internal ingresscontroller service", "service", desired)
 		return r.currentInternalIngressControllerService(ic)
 	case have:
-		// TODO Handle updates.
+		if updated, err := r.updateInternalService(current, desired); err != nil {
+			return true, current, fmt.Errorf("failed to update internal service: %v", err)
```

Comment:
Just confirming that this returns true because the internal service already exists and it doesn't matter if the update fails. True?

Reply:
True.  The Boolean return value only indicates whether the service exists.  I copied the pattern from other "ensure" methods, but I neglected to put that in the godoc.  I've now updated the godoc to explain this.